### ISSUE TITLE
bVNC feature request: auto-reconnect on e.g. loose connection

### ISSUE THREAD

**Issue by Aaron3142** [iiordanov/remote-desktop-clients#240]
An option to enable auto-reconnect after a broken connection would be extremely useful in cases where the network connection is unstable or when the server disconnects the clients after a given time.
I would suggest to enable the auto-retry only after a first successful connection (to avoid creating new issues when e.g. the credentials are not correct...)
Is this feature on the list you consider to implement (with high/low priority)?
Besides this little issue, great vnc-viewer!
Thank you
Aaron

**Comment by lypanov** [comment]
Fully agreed. Especially on the Boox Poke3 I'm running bVNC Pro on currently due to it's "hibernate on sleep" behavior I'm having to fully reconnect constantly.

On my Samsung Tab S6 I've been able to mostly work around this by pinning a widget with a specific connection. Alas on the Boox Poke3 the launcher doesn't support widgets so it's not an option. The ability to select a "default" connection with automatic reconnect would fix this for me.

Might just resort to hacking my defaults + reconnect into the source code and building an APK but need to find the time.
